docs/source/advanced/compiler-plugins.mdx

```diff
@@ -56,14 +56,14 @@ class MyPlugin: Plugin {
 }
 ```
 
-Make your plugin discoverable by [ServiceLoader](https://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html) using a resource in `src/main/resources/META-INF/services/com.apollographql.apollo3.compiler.Plugin`. This file contains the fully qualified name of your plugin: 
+Make your plugin discoverable by [ServiceLoader](https://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html) using a resource in `src/main/resources/META-INF/services/com.apollographql.apollo3.compiler.ApolloCompilerPlugin`. This file contains the fully qualified name of your plugin: 
 
 ```
 mypackage.MyPlugin
 ```
 
 > [!NOTE]  
```

Comment:
This doesn't seem to work as expected in [our docs](https://www.apollographql.com/docs/kotlin/v4/advanced/compiler-plugins/#:~:text=%5B!NOTE%5D,Plugins%20at%20runtime.)

Reply:
Good catch! This is GitHub markdown only indeed 👍